package.json

```diff
@@ -40,13 +40,14 @@
     "@types/uuid": "^8.3.0",
     "@typescript-eslint/eslint-plugin": "^4.13.0",
     "@typescript-eslint/parser": "^4.13.0",
+    "core-js": "^3.15.2",
```

Comment:
Why is this needed?

Reply:
It is needed for the tests to polyfill the `String.prototype.replaceAll` method (https://github.com/livestorm/livestorm-plugin/pull/16/files#diff-a1c786cdea90125d840669112d499d337a658bf6431a8c972a2e14301d908662R1)

Reply:
Not sure but I think you can call `String.prototype.replace` instead because the search argument is a regex with the `//g` flag. If so, this library should neither be needed.

Reply:
Indeed, it works with the `replace` method. `replaceAll` is better to use if the search arg is a string, this is not the case here so I will change to `replace` :)